receiver/solacereceiver/factory.go

```diff
@@ -25,6 +25,9 @@ import (
 
 const (
 	componentType config.Type = "solace"
+	// The stability level of the receiver.
+	stability = component.StabilityLevelAlpha
```

Comment:
I'd suggest the stability level to be "In Development".  The component was added this July and is not current in the Contrib distribution.

Reply:
Done.